### PR TITLE
feat(build): support building and executing on M1-based Macs

### PR DESCRIPTION
**Pull request by frrist**
Can build and run visor from my M1-Based MacBook, updates the README.md with steps to do so.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1695,8 +1698,9 @@ github.com/valyala/bytebufferpool v1.0.0 h1:GqA5TC/0021Y/b9FG4Oi9Mr3q7XYx6Kllzaw
 github.com/valyala/bytebufferpool v1.0.0/go.mod h1:6bBcMArwyJ5K/AmCkWv1jt77kVWyCJ6HpOuEn7z0Csc=
 github.com/valyala/fasttemplate v1.0.1 h1:tY9CJiPnMXf1ERmG2EyK7gNUd+c6RKGD0IfU8WdUSz8=
 github.com/valyala/fasttemplate v1.0.1/go.mod h1:UQGH1tvbgY+Nz5t2n7tXsz52dQxojPUpymEIMZ47gx8=
-github.com/valyala/gozstd v1.9.0 h1:Mse1yBM5ZQpigO4KW1bVK8p8VSRYttRF+pV0QsMWZ1Y=
 github.com/valyala/gozstd v1.9.0/go.mod h1:y5Ew47GLlP37EkTB+B4s7r6A5rdaeB7ftbl9zoYiIPQ=
+github.com/valyala/gozstd v1.11.0 h1:VV6qQFt+4sBBj9OJ7eKVvsFAMy59Urcs9Lgd+o5FOw0=
```
Comment (frrist) [inline go.sum:1702]: this is the real change, this is a dep of github.com/filecoin-project/go-bs-postgres-chainnotated and is required to build on M1 based systems.

**Review by iand — APPROVED** [review]
LGTM but should check it works with #544 since FFI has updated

**Comment by frrist** [comment]
Gosh I hate this linter

**Comment by frrist** [comment]
Well, it passes for me locally; nothing has changed with respect to the linter. If this ends up being an issue in subsequent PR's I'll dig in, but this failure seems benign.

**Pull request merged**